packages/ui/cypress/support/mount.ts

```diff
@@ -0,0 +1,31 @@
+import { mount } from '@cypress/vue'
+import type { Component } from 'vue'
+import { directives, plugins } from '../../client/global-setup'
+
+export const registerMount = () => Cypress.Commands.add(
+  'mount',
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  <C extends Parameters<typeof mount>[0]>(comp: any, options: any = {}) => {
```

Comment:
This C generic seems unused

Reply:
Yeah thanks for talking about this. I really need help learning how to best type these wrapper functions. Typescript is not my first language... 😓

Reply:
You should give a try of https://github.com/type-challenges/type-challenges!
In a stream, this could be fun to watch, especially with Anthony as mentor :P